pkg/cmd/admin/router/router.go

```diff
@@ -661,6 +650,17 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		if errs := bulk.Create(list, namespace); len(errs) != 0 {
 			return errExit
 		}
+
+		if needsSCC {
+			fmt.Fprintf(os.Stderr, "WARNING: Router service account %q is not privileged to open ports on the host IP\n", cfg.ServiceAccount)
```

Comment:
Do we want to also say `hey you need to add this router service account to the appropriate SCC` in order for the router to function properly. Being explicit is more helpful to the user.